hack/verify-bazel.sh

```diff
@@ -31,20 +31,24 @@ fi
 # TODO(spxtr): Remove this line once Bazel is the only way to build.
 rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
 
-# The git commit sha1s here should match the values in $KUBE_ROOT/WORKSPACE.
-kube::util::go_install_from_commit \
-    github.com/kubernetes/repo-infra/kazel \
-    e26fc85d14a1d3dc25569831acc06919673c545a
-kube::util::go_install_from_commit \
-    github.com/bazelbuild/rules_go/go/tools/gazelle/gazelle \
-    a280fbac1a0a4c67b0eee660b4fd1b3db7c9f058
-
-gazelle_diff=$(gazelle fix -build_file_name=BUILD,BUILD.bazel -external=vendored -mode=diff -repo_root="$(kube::realpath ${KUBE_ROOT})")
-kazel_diff=$(kazel -dry-run -print-diff -root="$(kube::realpath ${KUBE_ROOT})")
-
-if [[ -n "${gazelle_diff}" || -n "${kazel_diff}" ]]; then
-  echo "${gazelle_diff}"
-  echo "${kazel_diff}"
+_tmpdir="$(mktemp -d -t verify-bazel.XXXXXX)"
+function cleanup {
+  rm -rf "${_tmpdir}"
+}
+trap cleanup EXIT
```

Comment:
nit: could use `kube::util::trap_add`, but that's only required if we cascade scripts and need to call multiple traps.

Reply:
good suggestion, done.